src/runner/nunit.runner/Services/XmlFileProcessor.cs

```diff
@@ -92,13 +93,26 @@ public XmlFileProcessor(TestOptions options)
 
             for (int i = 0; i < segments.Length - 1; i++)
             {
-                IFolder folder = new FileSystemFolder(path);
-
-                var res = await folder.CheckExistsAsync(segments[i + 1]);
-                if (res != ExistenceCheckResult.FolderExists)
+                try
                 {
-                    await folder.CreateFolderAsync(segments[i + 1], CreationCollisionOption.OpenIfExists);
+#if __DROID__
+                    IFolder folder = await FileSystem.Current.GetFolderFromPathAsync(path, CancellationToken.None);
+#elif __IOS__
+                    IFolder folder = await FileSystem.Current.GetFolderFromPathAsync(path, CancellationToken.None);
+#else
+                    IFolder folder = await FileSystem.Current.GetFolderFromPathAsync(path.Replace('/', '\\'), CancellationToken.None);
```

Comment:
Just curious, did you do the path replacement because it wasn't working? I thought Windows accepted the Linux path separators, but I might be wrong.